README.md

```diff
@@ -1,6 +1,8 @@
-# CoronavirusForm
+# GOV.UK coronavirus - triage vulnerable people
```

Comment:
```suggestion
# GOV.UK coronavirus - find support service
```

I really like the name [suggested in this thread](https://gds.slack.com/archives/C010WFLGR4L/p1586160803001200?thread_ts=1586009941.002700&cid=C010WFLGR4L), could we use it for all non-repo names?

Reply:
I think we've got time to rename the repo if we want.

Reply:
Yeah, i really like find coronavirus - find support service, so long as it doesn't confuse with the shielded service stuff?

Shall we see what the powers that be say?

Reply:
Let's just JFDI it. I'll update the PR to use that name.